pkg/mounter/ossfs.go

```diff
@@ -246,37 +254,111 @@ func CleanupOssfsCredentialSecret(ctx context.Context, clientset kubernetes.Inte
 	return err
 }
 
-func buildAuthSpec(nodeName, volumeId string, authCfg *AuthConfig,
+func buildAuthSpec(nodeName, volumeId, target string, authCfg *AuthConfig, rrsaCfg *RrsaConfig,
 	spec *corev1.PodSpec, container *corev1.Container, options *[]string) {
 	if spec == nil || container == nil || options == nil {
 		return
 	}
-	if authCfg != nil && authCfg.AuthType == AuthTypeSTS {
-		return
+	if authCfg == nil {
+		authCfg = &AuthConfig{}
 	}
-	passwdMountDir := "/etc/ossfs"
-	passwdFilename := "passwd-ossfs"
-	passwdSecretVolume := corev1.Volume{
-		Name: "passwd-ossfs",
-		VolumeSource: corev1.VolumeSource{
-			Secret: &corev1.SecretVolumeSource{
-				SecretName: OssfsCredentialSecretName,
-				Items: []corev1.KeyToPath{
-					{
-						Key:  fmt.Sprintf("%s.%s", nodeName, volumeId),
-						Path: passwdFilename,
-						Mode: pointer.Int32Ptr(0600),
+
+	switch authCfg.AuthType {
+	case AuthTypeSTS:
+	case AuthTypeRRSA:
+		if rrsaCfg == nil {
+			return
+		}
+		spec.ServiceAccountName = authCfg.ServiceAccountName
+		rrsaMountDir := "/var/run/secrets/ack.alibabacloud.com/rrsa-tokens"
+		rrsaVolume := corev1.Volume{
+			Name: "rrsa-oidc-token",
+			VolumeSource: corev1.VolumeSource{
+				Projected: &corev1.ProjectedVolumeSource{
+					DefaultMode: tea.Int32(420),
```

Comment:
```suggestion
					DefaultMode: tea.Int32(0600),
```
Should this be more obvious?

Reply:
Just to be consistent with ack-pod-identity-webhook

Reply:
Then that one should be changed as well. 0600 is exactly the same as 420. 420 does not make sense in the context of file mode.